src/AppBar/AppBar.tsx

```diff
@@ -84,7 +84,9 @@ class ApplicationBar extends Component<IProps> {
                     </Paper>
                   }
                 </Grid>
-                <ExampleDocumentsDropdown examples={examples} onChange={onExampleDocumentsDropdownChange} />
+                {this.props.uiSchema && this.props.uiSchema.appBar && this.props.uiSchema.appBar["ui:examplesDropdown"] &&
```

Comment:
Perhaps we can make this into a function call instead? 

```
const shouldUseExampleDropdown = (uiSchema) => {
  return uiSchema && 
    uiSchema.appBar && 
    uiSchema.appBar["ui:examplesDropdown"];
};
```

or something like that anyways.